ui/app/components/auth-jwt.js

```diff
@@ -1,5 +1,6 @@
 import Ember from 'ember';
 import { inject as service } from '@ember/service';
+// ARG NOTE: Once you remove outer-html after glimmerizing you can remove the outer-html component
```

Comment:
Would it be helpful to add this todo onto the excel spreadsheet?

Reply:
I've been adding comments on the components. You'll see them as little yellow dog ears on the components name. That's where I've been keeping comments, including this one.

Reply:
awesome - thank you!